ui/components/app/signature-request-siwe/signature-request-siwe.js

```diff
@@ -60,53 +60,117 @@ export default function SignatureRequestSIWE({
   const [agreeToDomainWarning, setAgreeToDomainWarning] = useState(false);
 
   /**
+   * WIP
+   *
    * @todo update MetaMetrics for onSign and onCancel
    * Currently, we're tracking the events similarily to how we track the events with the SignatureRequest component
    * in which the action names differ. We want to revisit this to add properties mentioned in the related, internal Slack thread.
    * We would also want to ensure that we are tracking the event appropriatedly should an error be thrown.
+   *
+   * Follow-up Questions:
+   * 1. Are we utilizing MetaMetrics sensitive_properties?
+   * 2. Are we capturing the "Signature Request" event, maybe under a different event name?
+   * 3. Should we spend time to include account_hardware_type? We might be able to create a selector to perform findKeyringForAddress
+   * 4. Should we add validation_errors? what is this in the context of transactions?
+   * 5. Should we add consistency between event tracking here and the SignatureRequestComponent?
```

Comment:
Answers:
1. no answer yet
2. no. We may want to add this to createRPCMethodTrackingMiddleware.js
3. yes. We should be able to access the account and hardware type from the TransactionsController to createRPCMethodTrackingMiddleware.js
4. no
5. yes. We actually plan to remove the trackEvents from this component and SignatureRequestComponent in favor of tracking these events in createRPCMethodTrackingMiddleware.js

Credits to @brad-decker for the answers to these questions and the direction moving forward